src/features.njk

```diff
@@ -145,7 +145,7 @@ sitemapPriority: 0.8
                         <label class="flex items-center text-center justify-center font-medium text-2xl text-gray-800 mt-5">FlowForge Cloud</label>
                         <p>
                             <div class="text-sm leading-6 mb-6 mt-6">
-                                A managed hosted version maintained and supported by FlowForge employees. 
+                                Our managed hosted solution, maintained and supported by the FlowForge team. Experience flexibility, scalability, and expert assistance to drive your business forward.
```

Comment:
@ZJvandeWeg I changed this one too, it was too short and seemed like it had less features than the other two.